source/educators/how-tos/create_section.rst

```diff
@@ -1,6 +1,32 @@
-Create a Section
-#################
+.. _Create a Section Video:
+
+######################
+Create a Section Video
+######################
 
 .. tags:: educator, how-to
 
 .. include:: reusable_content/create_section.txt
+
+
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```

Comment:
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
  :ref:`Course Outline` (concept)
  
 :ref:`Creating a New Course` (how-to)
 
 :ref:`Create a Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Subsection ` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

Reply:
Are you checking how pages render? You need spaces here otherwise they all render in a big block which is illegible. This is a common mistake in this PR and is why I want smaller changesets, so we can catch problems early.

Reply:
@srushe1 can you take a pass through all our pages and check if there are any remaining places where this is happening?